Uploading dSYM to correct path with native apps (#1411)
codemagic-ci-cd/codemagic-docs: content/knowledge-firebase/firebase-crashlytics-dsym-uploading.md

        fi
{{< /highlight >}}
 

> The above-mentioned **dsymPath** is Flutter specific and it could change depending on what platform the app is built. E.g., for React Native or Native iOS applications you might use the dsymPath as:

Reviewer: ```suggestion
The above-mentioned **dsymPath** is Flutter-specific and could change depending on what platform the app is built on. For example, in React Native or Native iOS applications you might use the dsymPath as:
```

Author: Done!
